src/CommitmentsV2/SFA.DAS.CommitmentsV2.UnitTests/Mapping/ResponseMappers/GetApprenticeshipResponseMapperTests.cs

```diff
@@ -56,5 +60,23 @@ public void EmploymentEndDateIsMappedCorrectly()
         {
             Assert.AreEqual(_source.FlexibleEmployment.EmploymentEndDate, _result.EmploymentEndDate);
         }
+
+        [Test]
+        public void RecognisePriorLearningIsMappedCorrectly()
+        {
+            Assert.AreEqual(true, _result.RecognisePriorLearning);
```

Comment:
Does this assume that the RecognisePriorLearning will always be true? I think it will always be true. But should you reference the _source.RecognisePriorLearning field

Reply:
updated